OpenRA.Mods.Common/Widgets/Logic/MultiplayerLogic.cs

```diff
@@ -106,6 +111,11 @@ public MultiplayerLogic(Widget widget, ModData modData, Action onStart, Action o
 			widget.Get<ButtonWidget>("BACK_BUTTON").OnClick = () => { Ui.CloseWindow(); onExit(); };
 			Game.LoadWidget(null, "GLOBALCHAT_PANEL", widget.Get("GLOBALCHAT_ROOT"), new WidgetArgs());
 
+			lanGameLocations = new List<BeaconLocation>();
+			lanGameProbe = new Probe(Server.MasterServerPinger.OpenRALANGame);
```

Comment:
Ok, though to not make it ugly but still maintainable, I would create e.g. Constants class where we could put any such constants with no clear place.